lib/index.js

```diff
@@ -682,6 +683,13 @@ var Client = module.exports = function(config) {
                     basic = new Buffer(this.auth.username + ":" + this.auth.password, "ascii").toString("base64");
                     headers["Authorization"] = "Basic " + basic;
                     break;
+                case "netrc":
+                    var auth = netrc()["api.github.com"];
```

Comment:
Actually never mind, for that one endpoint the user will just see the error message and add an entry for uploads.githubcom to their .netrc.